db/migrate/20200531195209_add_status_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrders < ActiveRecord::Migration[5.1]
+  def change
+    add_column :orders, :status, :string, default: "Pending"
+  end
+end
```

Comment:
I changed my migration on my end to match this to reduce merge conflicts. If we decide to convert it back to an integer, it would be an easy fix.